src/types/registration.types.ts

```diff
@@ -201,12 +201,17 @@ export const emptyRegistrationDate: RegistrationDate = {
 };
 
 export interface RegistrationPreview {
+  contributors?: Contributor[];
+  identifier?: string;
```

Comment:
denne skal sikkert ikke være optional?
```suggestion
  identifier: string;
```